### PR TITLE
fix textinput lag by separating input into draft component

### PR DESCRIPTION
**Pull request by naborisk**

### CODE REVIEW

File: src/components/Draft.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { useState, useContext } from "react";
+import { TextField, Box, Stack, Button } from "@mui/material";
+import { Sign } from '../util'
+import { ApplicationContext } from '../App';
+import SendIcon from '@mui/icons-material/Send';
+
+export function Draft(props: any) {
```
Comment (totegamma) [inline src/components/Draft.tsx:7]: could you define Draft Interface Type like other components please

**Review by totegamma — CHANGES_REQUESTED** [review]
Thank you for your PR!
Could you change following points please?
- remove pnpm.lock
- define Interface of Draft.tsx
- (optional) remove debug code in App.tsx

File: src/components/Draft.tsx (hunk 1 above)
Comment (naborisk) [reply src/components/Draft.tsx]: I've updated the code according to the requested change.

**Review by totegamma — APPROVED** [review]
LGTM

Thank you for your contribution❤️

**Pull request merged**